keyboards/cannonkeys/satisfaction75/satisfaction75.c

```diff
@@ -101,13 +102,65 @@ void raw_hid_receive( uint8_t *data, uint8_t length )
 				command_data[3] = (value >> 8 ) & 0xFF;
 				command_data[4] = value & 0xFF;
 			}
+      else if ( command_data[0] == id_oled_default_mode )
```

Comment:
This would be easier to read if you switched on command_data[0] and used the id_xxxxx enums as case statements, or maybe even put the resulting nested switch into another function.

Reply:
Yes - that's a very good point. I'll change this to be that.